index.html

```diff
@@ -24,15 +24,45 @@
   <body>
     <header class="header">
       <nav id="navbar" class="navbar">
-        <a href="">Kennedy Owusu</a>
-        <a id="menu-icon" href=""><img src="./images/nav-icons/nav-icon.png"></a>
+        <a class="navbar-title" href="">Kennedy Owusu</a>
         <ul>
           <li><a class="header-nav" href="">Portfolio</a></li>
           <li><a class="header-nav" href="">About</a></li>
           <li><a class="header-nav" href="">Contact</a></li>
           <li><a class="header-nav" href=""><img src="./images/nav-icons/mail-icon.png"></a></li>
         </ul>
```

Comment:
- [ ] Please consider following Aimal's advice on hiding the desktop menu, the problem lies in your media query!! 😱 
![image](https://user-images.githubusercontent.com/77069011/202028299-440762d7-0a45-40b5-8c68-5a5bce305b33.png)

You're applying those styles whenever the device's width is 375px or less, so when you use a, let's say, 376px wide device, you get to see the desktop menu as shown here:

![image](https://user-images.githubusercontent.com/77069011/202028631-92a29654-77ad-4465-9824-a4a75f51591b.png)

💡 _Hint: You don't have to use that max-width media query at all, if you define your styles at the top level, they're going to get applied until the screen is at least 768px wide (The media query breakpoint you defined for desktop devices)._

Please, consider fixing this one! 🤓 🛠️

Reply:
I have fixed the issue. I thing I forgot to push the changes I made.

On Firefox
![540-on-firefox](https://user-images.githubusercontent.com/32777408/202123138-84ae75dd-a2d8-4ccb-89a1-9d90ba93bd13.png)

On chrome
![max-540](https://user-images.githubusercontent.com/32777408/202123413-90a8ffc2-a40c-4a9d-80b5-afd9ba6fd2d5.png)